e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-service/change-your-answers-agent-service-charging-no-to-yes.spec.js

```diff
@@ -72,10 +72,9 @@ context(
           const expectedUrl = `${agentChargesUrl}#${FIELD_ID}-label`;
           cy.assertUrl(expectedUrl);
 
-          // TODO: EMS-3828 - renable
           cy.completeAndSubmitAgentChargesForm({
-            // fixedSumMethod: true,
-            percentageMethod: true,
+            fixedSumMethod: true,
+            percentageMethod: false,
```

Comment:
could this be defaulted to false maybe? then we don't need to add this line in some tests?

Reply:
It is defaulted to false, but I think we should be explicit, we already do this in several places